### ISSUE TITLE
Can Hecatomb be used for searching not only viruses but also bacteria, fungi, and mycoplasma?

### ISSUE THREAD

**Issue by Key-man-fromArchive** [shandley/hecatomb#91]
Is it possible to change or customize the database? If you have any ideas related to this, could you please provide some assistance?

**Comment by beardymcjohnface** [comment]
Yes you can substitute your own databases. Check out the MMSeqs documentation on downloading or creating your own databases: https://mmseqs.com/latest/userguide.pdf

At the moment, you would need primary and secondary aa and nt databases. For the aa databases, read the bit on Creating a seqTaxDB; it's pretty easy if you're using uniprot accessions/databases. For the nt database, here's an example of how we do it:
https://github.com/shandley/hecatomb/blob/v0.2.2/accessory/create_mmseqs_viral_nt_taxaDB.sh

If you didn't want our iterative search strategy (where we use the smaller aa and nt databases to prefilter, before putting the hits through the larger secondary databases) then that would require more extensive changes to the pipeline rules themselves. Was there something specific you wanted to try?
